### ISSUE TITLE
Windows Miner

### ISSUE THREAD

**Issue by mbxck** [pegnet/pegnet#111]
As discussed briefly on discord with D Chapman and Carl - we must have a credible, working Windows miner else we instantly exclude a lot of the decentralised mining community.

A compiled and released Windows miner should be command prompt based with a simple config/text file to enter/change variables prior to execution.

Miner should output current process to command window and also write logs to a text file.

**Comment by PaulSnow** [comment]
I believe we need a credible windows solution (something more than install inside a linux VM)

**Comment by WhoSoup** [comment]
Golang does a good job at being platform independent. The current codebase compiles and runs fine on Windows. The only reason there isn't a "windows miner" at the moment is because nobody has compiled one and uploaded it, but that's changing with the next release. (The current "release" on github is very outdated even for lunix)

tl;dr: there will be a windows miner

**Comment by Cahl-Dee** [comment]
This was provided during the release of pM1. Closing.
